### DISCUSSION TITLE
Tracking loss values for different properties

### DISCUSSION THREAD

**Discussion by bryanpiguave** [chemprop/chemprop#1232] [Q&A]
I am using chemprop for a multitask property regression. I would like to track the loss values for each property as I train the model. I tried the cli and it seems that logs from tensorboard only provide the overall train and val loss. 
Is there any implementation for track individual losses?

**Comment by JacksonBurns** [comment]
This isn't implemented in ChemProp right now and not readily achievable in a general way with the current setup for loss metrics. You can jerry-rig something by overwriting the `training_step` method when using ChemProp as a Python module, but again, not available via the CLI.
